### ISSUE TITLE
Extra rubocop:disable makes cop enabled

### ISSUE THREAD

**Issue by jonas054** [bbatsov/rubocop#2546]
In #1690, @HaleTom wrote

> I have seen redundant disable comments being flagged. What I don't get is that second one here is not flagged though... and it actually CAUSES a warning???!!!

```ruby
# make rubocop happy
class Passes
  # rubocop:disable Style/ClassVars
  @@class_var = 'no problems here'
end

# make rubocopy sad
class Offensive
  # Removing the next comment line makes for a happy rubocop - why?
  # rubocop:disable Style/ClassVars
  @@class_var = 'broken?'
end
```

**Comment by jonas054** [comment]
Removing the second `rubocop:disable` makes `Style/ClassVars` disabled from line 3 to the end of the file. This is normal. The bug is that the second `rubocop:disable Style/ClassVars` seems to make the first one ineffective.

**Comment by alexdowad** [comment]
Thanks
